Cargo.lock

```diff
@@ -4887,8 +4887,7 @@ dependencies = [
 [[package]]
 name = "sparkle"
 version = "0.1.10"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "84b935631ddcc08b257a331ac645cc6831e52a4ff55d4c430142575c6757a094"
+source = "git+https://github.com/mmatyas/sparkle?branch=read_pixels_raw#45d8bb263d87edcf6c2608fe14d717430025fdc2"
```

Comment:
Seems like you didn't make a PR to sparkle yet?

Reply:
Yes, this was just a commit, but I've just opened a PR here: https://github.com/servo/sparkle/pull/14